### ISSUE TITLE
cannot find file tikzlexer.lexer.cpp

### ISSUE THREAD

**Issue by peer23peer** [tikzit/tikzit#12]
Hi,

I love that you're porting to QT since I'm more familiar with QT then GTK.
I'm currently searching for a tikz drawing program that could work well with own tikz circuit library [PIDcircuitTikZ](https://github.com/peer23peer/PIDcircuitTikZ) and tikzit shows promise. However when I try to compile commit 768e097 . I get an error that `tikzlexer.lexer.cpp` can not be found. It could be that I'm missing something obvious since I'm unfamiliar with bison and flex.

```cmd
14:05:42: Running steps for project tikzit...
14:05:42: Starting: "C:\Qt\5.9.1\mingw53_32\bin\qmake.exe" D:\Projects\thirdparty\tikzit\tikzit.pro -spec win32-g++ "CONFIG+=debug" "CONFIG+=qml_debug"
Info: creating stash file D:\Projects\thirdparty\build-tikzit-Desktop_Qt_5_9_1_MinGW_32bit-Debug\.qmake.stash
14:05:45: The process "C:\Qt\5.9.1\mingw53_32\bin\qmake.exe" exited normally.
14:05:45: Starting: "C:\Qt\Tools\mingw530_32\bin\mingw32-make.exe" qmake_all
mingw32-make: Nothing to be done for 'qmake_all'.
14:05:46: The process "C:\Qt\Tools\mingw530_32\bin\mingw32-make.exe" exited normally.
14:05:46: Starting: "C:\Qt\Tools\mingw530_32\bin\mingw32-make.exe" 
C:/Qt/Tools/mingw530_32/bin/mingw32-make -f Makefile.Debug
mingw32-make[1]: Entering directory 'D:/Projects/thirdparty/build-tikzit-Desktop_Qt_5_9_1_MinGW_32bit-Debug'
flex --header-file -o ..\tikzit\src\data/tikzlexer.lexer.cpp ..\tikzit\src\data\tikzlexer.l
Makefile.Debug:217: recipe for target '../tikzit/src/data/tikzlexer.lexer.cpp' failed
mingw32-make[1]: Leaving directory 'D:/Projects/thirdparty/build-tikzit-Desktop_Qt_5_9_1_MinGW_32bit-Debug'
process_begin: CreateProcess(NULL, flex --header-file -o ..\tikzit\src\data/tikzlexer.lexer.cpp ..\tikzit\src\data\tikzlexer.l, ...) failed.
make (e=2): The system cannot find the file specified.

mingw32-make[1]: *** [../tikzit/src/data/tikzlexer.lexer.cpp] Error 2
Makefile:36: recipe for target 'debug' failed
mingw32-make: *** [debug] Error 2
14:05:46: The process "C:\Qt\Tools\mingw530_32\bin\mingw32-make.exe" exited with code 2.
Error while building/deploying project tikzit (kit: Desktop Qt 5.9.1 MinGW 32bit)
When executing step "Make"
14:05:46: Elapsed time: 00:04.
```

**Comment by akissinger** [comment]
Hi Jelle,

I've had some trouble with this as well. Something appears to be not exactly right with way qmake is set up for flex/binson. Are you building with qmake at the console, or are you using something like Qt Creator?

**Comment by peer23peer** [comment]
I'm building the project with Qt creator.

**Comment by peer23peer** [comment]
I tried to compile on my personal computer running Arch linux and it compiled without complaint. The issue seemed to be with Qt Creator and windows. I'm reminded yet again why I prefer Linux over Windows.

Luckily the problem was easily solved by downloading [win_flex_bison](https://sourceforge.net/projects/winflexbison/files/win_flex_bison-2.5.5.zip/download), renaming the `win_flex.exe` to `flex.exe` and `win_bison.exe` to `bison.exe`. Finally I added the installation directory to the PATH. 
It doesn't complain about flex and bison anymore.
